drivers/volume/portworx/schedops/k8s-schedops.go

```diff
@@ -451,6 +451,18 @@ func (k *k8sSchedOps) UpgradePortworx(ociImage, ociTag string) error {
 	return nil
 }
 
+//Method to validate if Portworx pod is up and running
+func (k *k8sSchedOps) IsPXPodRunningOnNode(n node.Node) bool {
+	pxPods, _ := k8s.Instance().GetPodsByNode(n.Name, "kube-system")
+	for _, pod := range pxPods.Items {
+		if strings.Contains(pod.Name, "portworx") && pod.Status.Phase != "Running" {
```

Comment:
nit: we can use strings.HasPrefix to be a little more strict

Reply:
Fixed.